domains/franciscovmag.json

```
@@ -4,6 +4,6 @@
         "email": "franciscovmag@hotmail.com"
     },
     "records": {
-        "CNAME": "ynk0.github.io"
+        "CNAME": "ynk0.github.io/portafolio/"
```

```suggestion
        "CNAME": "ynk0.github.io"
```
`CNAME` should be a domain name.
[Please read the docs.](https://docs.is-a.dev/guides/github-pages/)
If you cant "connect" your github pages website with `franciscovmag.is-a.dev`, open your github pages website repository settings > pages, and enter `franciscovmag.is-a.dev` in Custom Domain input. Then wait for DNS check and check Enforce HTTPS.